### ISSUE TITLE
Missing Image for UI.SetGlobalProperties

### ISSUE THREAD

**Issue by JackLivio** [smartdevicelink/sdl_hmi_integration_guidelines#78]
Missing `SetGlobalProperties_TTS_UI_VR.png`

**Comment by JackLivio** [comment]
Removing this code from page for now until asset is found
```
|||
SetGlobalProperties for active app within TTS.SetGlobalProperties_request, UI.SetGlobalProperties_request, VR.Started, VR.Stopped.   
![SetGlobalProperties](./assets/SetGlobalProperties_TTS_UI_VR.png)
|||
```

**Comment by jacobkeeler** [comment]
Seems that this is the image in question: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/feature/set_global_properties_keyboard_properties_struct/docs/UI/SetGlobalProperties/assets/SetGlobalProperties_TTS_UI_VR.png

**Comment by theresalech** [comment]
Closed via #205
